caput/mpiarray.py

```diff
@@ -83,39 +88,39 @@
 Print the third column of the array on all ranks
 
 >>> for ri in range(mpitools.size):
-...    if ri == mpitools.rank:
-...        print(ri, arr2)
-...    mpitools.barrier()
+...     if ri == mpitools.rank:
+...         print(ri, arr2)
+...     mpitools.barrier()
 0 [0.]
 
 Fetch a view of the whole array with a partial slice. The final two ranks should be None
```

Comment:
```suggestion
Fetch a view of the whole array with a partial slice. The final two ranks should be None

```
Blank line needed to properly format the code block.